server/cmd/dcrdex/config.go

```diff
@@ -40,7 +40,9 @@ const (
 	defaultRPCHost             = "127.0.0.1"
 	defaultRPCPort             = "7232"
 
-	defaultRegFeeConfirms   = 4
+	// NOTE: If we are not using an asset variable here, we need to add this to
+	// the config response and update the spec.
+	defaultRegFeeConfirms   = 1 // 4 is the default setting.
```

Comment:
This should be added to the the `ConfigResult` struct. 

@chappjc If you agree, can you update #130?

Reply:
Agree, I'll put it as a dex-wide setting.

Reply:
Now included in https://github.com/decred/dcrdex/pull/130